compute_tools/src/spec.rs

```diff
@@ -476,6 +484,11 @@ pub fn handle_databases(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
                 query.push_str(&db.to_pg_options());
                 let _guard = info_span!("executing", query).entered();
                 client.execute(query.as_str(), &[])?;
+                let grant_query: String = format!(
+                    "GRANT ALL PRIVILEGES ON DATABASE {} TO neon_superuser",
```

Comment:
Don't know / remember what `ALL PRIVILEGES` means in this context, so some comments would be useful. cc @lubennikovaav